### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by johananl**
**Release note**:

```other operator
Terraformer has been upgraded to v2.1.0.
```

### CODE REVIEW

**Comment by gardener-robot-ci-2** [comment]
Thank you @johananl for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: charts/images.yaml
Comment (rfranzke) [inline charts/images.yaml:33]:
```suggestion
- name: csi-attacher
  sourceRepository: https://github.com/kubernetes-csi/external-attacher
  repository: quay.io/k8scsi/csi-attacher
  tag: v3.1.0
  runtimeVersion: ">= 1.18"
```

File: charts/images.yaml
Comment (rfranzke) [inline charts/images.yaml:53]:
```suggestion
- name: csi-node-driver-registrar
  sourceRepository: https://github.com/kubernetes-csi/node-driver-registrar
  repository: quay.io/k8scsi/csi-node-driver-registrar
  tag: v2.1.0
  runtimeVersion: ">= 1.18"
```

File: charts/images.yaml
Comment (rfranzke) [inline charts/images.yaml:73]:
```suggestion
- name: csi-provisioner
  sourceRepository: https://github.com/kubernetes-csi/external-provisioner
  repository: quay.io/k8scsi/csi-provisioner
  tag: v2.1.0
  runtimeVersion: ">= 1.18"
```

File: charts/images.yaml
Comment (johananl) [reply charts/images.yaml]: Ah, great. Didn't know this was possible :slightly_smiling_face:

File: charts/images.yaml
Comment (rfranzke) [inline charts/images.yaml:5]: We just released [Terraformer v2.1.0](https://github.com/gardener/terraformer/releases/tag/v2.1.0), so there is now a Packet-specific image that only contains the TF Packet provider plugin, hence, is much smaller! See also https://github.com/gardener/gardener-extension-provider-aws/pull/241

```yaml
- name: terraformer
  sourceRepository: https://github.com/gardener/terraformer
  repository: eu.gcr.io/gardener-project/gardener/terraformer-packet
  tag: v2.1.0
```

Comment (johananl) [reply charts/images.yaml:5]: Cool, will update and test.

Comment (johananl) [reply charts/images.yaml]: LGTM. I've pushed the update.

**Comment by johananl** [comment]
We will likely run into the year change in the license again. Maybe we should wait for #100 and then rebase over `master`.

**Comment by johananl** [comment]
Actually let's wait with merging this anyway. I might have to push another change around the CSI-related images.

**Comment by rfranzke** [comment]
Already, then
/hold for now
Please `/unhold` when ready!

**Comment by timebertt** [comment]
Ah nice, I see you already included the terraformer upgrade, just wanted to do this :)
Can you add a release note about it, too?

**Comment by johananl** [comment]
> Ah nice, I see you already included the terraformer upgrade, just wanted to do this :)
> Can you add a release note about it, too?

Done!

**Comment by rfranzke** [comment]
@johananl if you are ready please comment with `/unhold` :)

**Comment by johananl** [comment]
Yes, thanks :-) I'm waiting for #103 (because it doesn't make sense to me to leave the CSI-related images outdated here but I also don't want to update the images and then remove them). Once #103 is merged I'll rebase this PR and update only the remaining images.

**Review by rfranzke — APPROVED** [review]
/lgtm
/reviewed ok-to-test

File: charts/images.yaml
Comment (rfranzke) [inline charts/images.yaml:3]: Hmm, it seems the https://concourse.ci.gardener.cloud/builds/289964 step does not like these `https://` :(

Comment (johananl) [reply charts/images.yaml:3]: Huh. No problem, I can remove those. But is the CI selective about them? Note that *some* of the images have `https://` while others don't:

- https://github.com/gardener/gardener-extension-provider-packet/blob/49af05c8218229b8ab7104244f33cf72cc87ec30/charts/images.yaml#L7
- https://github.com/gardener/gardener-extension-provider-packet/blob/49af05c8218229b8ab7104244f33cf72cc87ec30/charts/images.yaml#L11

I just wanted to tidy this file up. Will undo my changes for now.

Comment (johananl) [reply charts/images.yaml]: Actually I removed the `https://` from everything. Let's see if this is better :slightly_smiling_face:

**Review by rfranzke — APPROVED** [review]

**Pull request merged**
